src/components/List.js

```diff
@@ -114,26 +151,31 @@ const List = ({ token }) => {
                 ?.toLowerCase()
                 ?.includes(searchInput.toLowerCase().trim()),
             )
-            ?.map((doc) => {
+            ?.map((sortedList) => {
               return (
-                <li key={doc.id}>
+                <li
+                  key={sortedList.id}
+                  className={backgroundColor(sortedList)}
+                  aria-label={backgroundColor(sortedList)}
+                >
                   <input
                     type="checkbox"
                     aria-label="purchased-checkbox"
-                    name={doc.itemName}
-                    id={doc.id}
+                    name={sortedList.itemName}
+                    id={sortedList.id}
                     onChange={handleCheckbox}
-                    checked={checkPurchasedDate(doc.lastPurchased)}
-                    disabled={checkPurchasedDate(doc.lastPurchased)}
+                    checked={checkPurchasedDate(sortedList.lastPurchased)}
+                    disabled={checkPurchasedDate(sortedList.lastPurchased)}
                   />
-                  {doc.itemName}
-                  {doc.numberOfPurchases > 0 ? (
+                  {sortedList.itemName}
+                  {sortedList.numberOfPurchases > 0 ? (
                     <p>
-                      Time until next purchase: {getDaysUntilNextPurchase(doc)}{' '}
-                      days
+                      Time until next purchase:{' '}
+                      {getDaysUntilNextPurchase(sortedList)} days. Purchased{' '}
```

Comment:
We added this UI in for now to help us check that it's sorting the list as we intended.